packages/core/src/external/fhir/patient/index.ts

```diff
@@ -67,6 +68,14 @@ export function toFHIR(patient: PatientIdAndData): FHIRPatient {
   };
 }
 
+export function toFHIR(patient: PatientIdAndData): FHIRPatient {
+  return mapPatientDataToResource(patient);
+}
+
+export function toIheGatewayPatientResource(patient: PatientIdAndData): PatientResource {
```

Comment:
added a function that uses the ihe-gateway-sdk patientResource type. The FHIR type has a bunch of optional fields that are actually required, namely "Name" and "DOB". Those are required fields in our PatientData definition, but because we were conveting our PatientData object to FHIR Patient with all those fields being optional, and then treating it like they were required, the situation was kinda a mess. Now we are using the same PatientResource type throughout the flow which reflects the fields that are actually required in the IHE profile

Reply:
I see, it makes sense. I'd just place `toIheGatewayPatientResource` on a IHE GW v2 related folder, not FHIR.